### ISSUE TITLE
Running a process in Terminal shows full process invocation including environment variables

### ISSUE THREAD

**Issue by cdvillard** [zed-industries/zed#8641]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

When running a process in the terminal, the tab name will change and show the entire invoked command complete with environment variables and all values involved in the calculated command called. For example, when running `npm run start` in a particular project, the terminal will show sensitive information in the tab if I were to hover. I imagine this could prove a potential security issue.
![Screenshot 2024-02-29 at 6 16 07 PM](https://github.com/zed-industries/zed/assets/6250214/a3de8eb4-9874-4b40-a28a-82e6bedf6360)

### Environment

Zed: v0.124.7 (Zed)
OS: macOS 14.1.2
Memory: 16 GiB
Architecture: x86_64

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.
If you only need the most recent lines, you can run the `zed: open log` command palette action to see the last 1000.

_No response_

**Comment by dalton-oliveira** [comment]
@JosephTLyons I'm working on a PR #8998 that touches this part. Can get this after that but I'm unsure what's the expected behavior here. Is it to never show any command arguments ?

**Comment by mikayla-maki** [comment]
Perhaps we shoud integrate this with the `redact_private_values` variable and simply remove everything in the toolbars?

**Comment by cdvillard** [comment]
@dalton-oliveira speaking for myself, I'm uses to just seeing the base command. For example, in the above example, instead of `ui - node etc. etc.` it can just show `ui - node`. To maintain the UX, this could be set behind a settings toggle. I'm also wondering if this could be built in a way where hovering over the tab while holding option would reveal the unredacted tool tip, @mikayla-maki?

**Comment by DemianParkhomenko** [comment]
This issue still present, is there any way to hide env variables or at least make terminal tab name smaller? 🙂  

<img width="392" alt="image" src="https://github.com/user-attachments/assets/4da9b8ff-30cd-4f3b-afb4-8304afa13a8b">

@dalton-oliveira @mikayla-maki

**Comment by maxbrunsfeld** [comment]
I agree; since environment variables can contain sensitive information, we should probably err on the side of not displaying them in the tab. Also, there's limited space in the tab, so this might sometimes help to show information that is more useful, like the command  itself and its arguments.

**Comment by notpeter** [comment]
I believe this has been fixed. I no longer see environment variables for a terminal/task in the UI.

If you are still seeing this issue, please comment below with details for how to reproduce and I'm happy to reopen.

Thanks
